src/sdk/models/policyStatistics.ts

```diff
@@ -0,0 +1,36 @@
+import { IEntityIdentifier } from './base';
+
+export interface PolicyStatistic {
+	name?: string;
+	description?: string;
+	bytes?: number;
+	files?: number;
+	folders?: number;
+	versions?: number;
+	storage?: number;
+	cost_savings?: number;
+}
+
+export interface PolicyStatisticsList {
+	[key: string]: PolicyStatistic;
+}
+
+export type PolicyJobStatisticsTimeline = { 
+	timestamp: number; 
+	stats: {
+		by_risk?: PolicyStatisticsList;
+		by_group?: PolicyStatisticsList;
+		by_activity?: PolicyStatisticsList;
+	}
+}[];
+
+export interface PolicyJobStatistics extends IEntityIdentifier<string> {
+	timeline?: PolicyJobStatisticsTimeline;
+	by_risk?: PolicyStatisticsList;
+	by_group?: PolicyStatisticsList;
+	by_activity?: PolicyStatisticsList;
+	by_location_health?: PolicyStatisticsList;
```

Comment:
We won't have statistics by health.

Reply:
How will the `locations` pie chart be populated in the `Policy overview`? https://www.figma.com/proto/54xvnM968kcVTAKfUTjc8X/Policy-Engine?node-id=174%3A751&scaling=min-zoom

Reply:
Ugh, I guess we will need to calculate then.